server/test/businesses.js

```diff
@@ -9,7 +9,7 @@ chai.use(chaiHttp);
 
 process.env.NODE_ENV = 'test';
 
-const token = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySWQiOjEsInVzZXJuYW1lIjoibW9zZXMiLCJlbWFpbCI6ImhlaWdodEB3aWR0aC5jb20iLCJpYXQiOjE1MjIyMTM4MDYsImV4cCI6MTUyMjI0OTgwNn0.1VoncNl5gRtoXx5zHo3MZZ9ev41Pi3f-CD9AY8Qw5OU'
+const token = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySWQiOjEsInVzZXJuYW1lIjoibW9zZXMiLCJlbWFpbCI6ImhlaWdodEB3aWR0aC5jb20iLCJpYXQiOjE1MjIyOTc5NjcsImV4cCI6MTUyMjMzMzk2N30.UjgIqVQ67Pn4N3ZIeYpiQE--028EBc79a0NlofFriqg'
```

Comment:
Hi Samson, great job you have done so far.  I am seeing token here, I suggest you put it in a ".env" file for security reasons. Please kindly look it up.

Reply:
Thanks for this. I will endeavour to fix this and stop it from happening agasin.